src/py/_pyodide/_core_docs.py

```diff
@@ -466,7 +466,7 @@ class Promise(JsProxy):
     pass
 
 
-def destroy_proxies(pyproxies: JsProxy, /):
+def destroy_proxies(pyproxies: JsProxy, /) -> Any:
```

Comment:
```suggestion
def destroy_proxies(pyproxies: JsProxy, /) -> None:
```
https://github.com/pyodide/pyodide/blob/af972267c7969c6f023da709851463ec99e7e4ac/src/core/python2js.c#L838